test/core/tsi/crl_ssl_transport_security_test.cc

```diff
@@ -54,103 +53,88 @@ const char* kSslTsiTestCrlSupportedCrlDirMissingIntermediate =
 const char* kSslTsiTestCrlSupportedCrlDirMissingRoot =
     "test/core/tsi/test_creds/crl_data/crls_missing_root/";
 const char* kSslTsiTestFaultyCrlsDir = "bad_path/";
+const std::string kRevokedKeyPath =
+    absl::StrCat(kSslTsiTestCrlSupportedCredentialsDir, "revoked.key");
+const std::string kRevokedCertPath =
+    absl::StrCat(kSslTsiTestCrlSupportedCredentialsDir, "revoked.pem");
+const std::string kValidKeyPath =
+    absl::StrCat(kSslTsiTestCrlSupportedCredentialsDir, "valid.key");
+const std::string kValidCertPath =
+    absl::StrCat(kSslTsiTestCrlSupportedCredentialsDir, "valid.pem");
+const std::string kRevokedIntermediateKeyPath = absl::StrCat(
+    kSslTsiTestCrlSupportedCredentialsDir, "leaf_signed_by_intermediate.key");
+const std::string kRevokedIntermediateCertPath = absl::StrCat(
+    kSslTsiTestCrlSupportedCredentialsDir, "leaf_and_intermediate_chain.pem");
+const std::string kRootCrlPath =
+    absl::StrCat(kSslTsiTestCrlSupportedCrlDir, "current.crl");
+const std::string kIntermediateCrlPath =
+    absl::StrCat(kSslTsiTestCrlSupportedCrlDir, "intermediate.crl");
+
+std::string LoadFile(absl::string_view file_path) {
```

Comment:
Please use the existing [`GetFileContents()` function](https://github.com/grpc/grpc/blob/7fabc61f07be423cccda79463f30dc390b83ac9c/test/core/util/tls_utils.h#L60) instead of duplicating this functionality.